microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -904,30 +948,98 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 125.
-* This checkbox is present in Microsoft Edge Stable 124.
+* This checkbox is not present in Microsoft Edge Canary 128.
+* This checkbox is present in Microsoft Edge Stable 125.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable user timings based extensions
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new label for checkbox as of Microsoft Edge 128_
+
+Enables extensions that customize the appearance of user timing events in the **Performance** tool.
+
+See also:
+* [Performance features reference](../evaluate-performance/reference.md)
+* [Performance recording event reference](../evaluate-performance/performance-reference.md)
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 128.
+* This checkbox is present in Microsoft Edge Stable 125.
 
 
 <!-- ====================================================================== -->
 ## Timeline: Enable User Timings based extensions
-<!-- no (?) link -->
+<!-- checkbox has no (?) link pointing to this anchor wording -->
 
 _new as of Microsoft Edge 125_
 
-Enables extensions that customize the appearance of user timing events in the **Performance** tool.
+See the above section, showing the newer label for the checkbox.
+
+
+<!-- ====================================================================== -->
+## Performance panel: Enable collecting enhanced traces
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new as of Microsoft Edge 128_
+
+In the **Performance** tool, when recording, collects enhanced traces.
+<!-- todo: real desc - maybe extract from article -->
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 128.
+* This checkbox is not present in Microsoft Edge Stable 125.
 
 See also:
+* [Share enhanced performance and memory traces](../experimental-features/share-traces.md)
 * [Performance features reference](../evaluate-performance/reference.md)
 * [Performance recording event reference](../evaluate-performance/performance-reference.md)
 
+
+<!-- ====================================================================== -->
+## Performance panel: Enable collecting source text for compiled script
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new as of Microsoft Edge 128_
+
+In the **Performance** tool, when recording, collects source text (the original source code) for compiled JavaScript code.
+<!-- todo: real desc -->
```

Comment:
Performance panel: Enable collecting source text for compiled script
Need real desc @captainbrosset
check https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/share-traces?branch=pr-en-us-3192

Reply:
My limited understanding of this experiment is that it doesn't have an impact on the UI of DevTools. It's used to make it possible to store the source code of compiled scripts in the trace file, when exporting it.

@robpaveza: can you please help with this one too? Is this used when loading an enhanced trace in a new window, perhaps?

Reply:
```
In the **Performance** tool, when exporting the trace file, stores the source code of 
compiled scripts in the trace file.  <!-- This experiment is used when loading an enhanced 
trace in a new window. -->
```

Reply:
Jayson to assist

Reply:
Same as the other one, this experiment is for performance panel capturing newly added traces for script source text to support enhanced traces v2 (currently implementing in upstream). It does not affect the ability of downstream enhanced traces and the state of it doesn't impact downstream user's experience.

Reply:
result:
```
In the **Performance** tool, when exporting the trace file, stores the source code of 
compiled scripts in the trace file.

This experiment makes the **Performance** tool capture newly added traces for script 
source text, to support enhanced traces v2.  This feature is being implemented in 
upstream.  This experiment doesn't affect the capabilities of downstream enhanced 
traces.  The state of this checkbox doesn't impact a downstream user's experience.
```